pallets/offchain-worker/src/lib.rs

```diff
@@ -287,7 +295,7 @@ pub mod pallet {
 		fn clear_claim() {
 			// Remove all account index in last session
 			<ClaimAccountIndex<T>>::remove_all();
-
+// SBP M1: can't this be replaced by a single map?
```

Comment:
single map also works. all accounts in the container will be processed. can't see any benefit to put the data in the map.

Reply:
That's a single structure to maintain, so less error prone

Reply:
Actually, we use the vector for other reason. I hope the ocw index in next round of data querying could be different and random. So we collect data from map and then put them into vector. At last, use the index of ocw account in the vector as the index of ocw tasks.

Reply:
:+1: